okta/api_response.py

```diff
@@ -14,6 +14,7 @@ def __init__(self, request_executor, req, res_details, response_body="",
                  data_type=None):
         self._url = res_details.url
         self._headers = req["headers"]
+        self._resp_headers = res_details.headers
```

Comment:
Since "api_response" represents the response, why `_heathers` prop doesn't contain all the heathers the server sends? Should we just add missing headers to `_headers`?

Reply:
_headers - are the request headers, which are used for a new request in case of pagination
_resp_headers - response headers